docs/gettingstarted/quickstart.rst

```diff
@@ -0,0 +1,268 @@
+      | Links, ports and volumes work across hosts                               |
+      +--------------------------------------------------------------------------+
+      | After deployment, containers can move around                             |
+      +--------------------------------------------------------------------------+
+
+      .. empty-div:: arrow-down center-block
+
+   .. parallel::
+
+      .. mobile-label::
+
+         Local
+
+
+      Installation
+      ------------
+
+      .. noscript-content::
+      
+         OS X
+         ^^^^
+
+         Install the flocker-cli client on your Mac (requires Homebrew):
+
+         .. code-block:: console
+
+            $ brew update && \
+              brew tap clusterhq/flocker && \
+              brew install flocker-0.3.2
+            
+      .. noscript-content::
+
+         Linux
+         ^^^^^
+
+         Install the flocker-cli client on your Linux machine:
+
+         .. code-block:: console
+
+            $ virtualenv flocker-tutorial && \
+              flocker-tutorial/bin/pip install --upgrade pip && \
+              flocker-tutorial/bin/pip install --quiet https://archive.clusterhq.com/downloads/flocker/Flocker-0.3.2-py2-none-any.whl
+
+      .. tabs::
+
+         OS X
+         ^^^^
+
+         Install the flocker-cli client on your Mac (requires Homebrew):
+
+         .. code-block:: console
+
+            $ brew update && \
+              brew tap clusterhq/flocker && \
+              brew install flocker-0.3.2
+
+         Linux
+         ^^^^^
+
+         Install the flocker-cli client on your Linux machine:
+
+         .. code-block:: console
+
+            $ virtualenv flocker-tutorial && \
+              flocker-tutorial/bin/pip install --upgrade pip && \
+              flocker-tutorial/bin/pip install --quiet https://archive.clusterhq.com/downloads/flocker/Flocker-0.3.2-py2-none-any.whl
+
+      .. empty-div:: arrow-down center-block invisible
+
+   .. parallel::
+
+      .. mobile-label::
+
+         Live
+
+
+      Installation
+      ------------
+
+      .. noscript-content::
+
+         Vagrant
+         ^^^^^^^
+
+         Simulate a Flocker cluster with virtual machines on your laptop (requires Vagrant, VirtualBox):
+
+         .. code-block:: console
+
+            $ git clone \
+              https://github.com/clusterhq/vagrant-flocker && \
+              cd vagrant-flocker && \
+              vagrant up
+
+         AWS
+         ^^^
+
+         Please see our separate `AWS install instructions <http://docs.clusterhq.com/en/latest/gettingstarted/installation.html#using-amazon-web-services>`_ to get started.
+
+      .. tabs::
+
+         Vagrant
+         ^^^^^^^
+
+         Simulate a Flocker cluster with virtual machines on your laptop (requires Vagrant, VirtualBox):
+
+         .. code-block:: console
+
+            $ git clone \
+              https://github.com/clusterhq/vagrant-flocker && \
```

Comment:
installing VirtualBox on Fedora is supposedly like:

```
wget -q https://www.virtualbox.org/download/oracle_vbox.asc
rpm --import oracle_vbox.asc
curl http://download.virtualbox.org/virtualbox/rpm/fedora/virtualbox.repo > /etc/yum.repos.d/virtualbox.repo
yum install VirtualBox-4.3
```

However, while this gave me a working `VBoxManage` command it didn't give me a working virtualbox backend for vagrant...